assets/js/translate-analytics.js

```diff
@@ -0,0 +1,42 @@
+// Detects if the page is translated and notifies analyticis
+const getNewLanguage = () => {
+  const html = document.getElementsByTagName("html")[0];
+  return html.getAttribute("lang")?.valueOf();
+};
+
+// adapted from: https://analytical42.com/2022/detect-track-translations-ga4/
+export default () => {
+  // Start by checking if the MutationObserver API is available
+  if (typeof MutationObserver === "function") {
+    // Tell the observer to monitor for changes to HTML attributes
+    const config = { attributes: true };
+    // Build the function to run when a change is observed
+    const callback = mutationList => {
+      // Loop through each observed change
+      for (let i = 0; i < mutationList.length; i += 1) {
+        // Only do something if the change was on an attribute
+        if (mutationList[i].type === "attributes") {
```

Comment:
Sorry I forgot to send you this earlier, [here's the other code that deals with mutations](https://github.com/mbta/dotcom/blob/master/assets/ts/ui/accordion.ts#L10)... so these can have a property called `oldValue`, that you can read if interested in the old value.

Reply:
What we want is the new value of the lang attribute in the `html` element.  The helper code above gets the current one for us.